js/main.js

```diff
@@ -1 +1,57 @@
-/* Javascript goes here! */
+console.log('hello');/* Javascript goes here! */
+
+
+$(document).ready(function() {
+  $('.btn-primary').on('click', requestWeatherData);
```

Comment:
Instead of having to add a class to you can select this specific input, you could also do a form submit event listener.